### ISSUE TITLE
The context menu disappears

### ISSUE THREAD

**Issue by gryffcom** [vctfence/scrapbee#100]
I have the problem after updating Scrapbee. The context menu disappears. What should I do to get the context menu back?
Some information about my system:

~ % uname -a
FreeBSD india 13.2-RELEASE-p8 FreeBSD 13.2-RELEASE-p8 GENERIC amd64
I am using scrapbee_backend_bsd_64.

[info] version = 2.6.1
[info] browser = Firefox 121.0
[info] platform = Linux x86-64
[info] start web server: port = '9000'. pwd = 'null'
[info] connect web server: connected, version = 1.7.7
[info] show rdf list
[info] switch to rdf "/home/gryff/.app_data/scrapbee/scrapbee1.rdf"
[info] rdf loaded in 818ms

2024/01/22 12:33:43 scrapbee_backend.go:489: Start backend
2024/01/22 12:33:44 scrapbee_backend.go:550: Message: command = web-server
2024/01/22 12:33:44 scrapbee_backend.go:219: Create web server on address 127.0.0.1:9000
2024/01/22 12:33:44 scrapbee_backend.go:219: Start Server and hold
2024/01/22 12:33:48 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/scrapbee1.rdf 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20231003095549657680/favicon.ico 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20230223131635749143/favicon.ico 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20240118123248396584/favicon.ico 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20231211095228162178/favicon.ico 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20231223195853554277/favicon.ico 
2024/01/22 12:33:51 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20240103202906630443/favicon.ico 
2024/01/22 12:33:52 scrapbee_backend.go:355: name /home/gryff/.app_data/scrapbee/data/20240106194057175560/favicon.svg

**Comment by Kerenok** [comment]
I too miss the context menu (right click on a page)!

**Comment by t1onix** [comment]
Right, missing functionality!  I add this to a truly strange and dysfunctional behavior for which I already made a report.
